system/metrics/sinks/log/logSink.cpp

```diff
@@ -69,21 +76,37 @@ void LogMetricSink::writeLogEntry(const std::shared_ptr<IMetric> &pMetric)
         size_t countBucketValues = values.size();
         __uint64 cumulative = 0;
 
+        // Accumulate values to output only if measurements have been taken
         for (size_t i=0; i < countBucketValues - 1; ++i)
```

Comment:
trivial.  Simpler to loop until < countBucketValues and avoid the extra increment after the loop.